rtf_vis_tool/src/Components/FrontendSummary.js

```diff
@@ -0,0 +1,63 @@
+/* Component to render Frontend Summary metrics. Specifically rotation sucesses, translation
+successes, pose successes, and correspondence inliers.
+
+Author: Adi Singh
+*/
+import React from "react";
+
+// Local Imports.
+import '../stylesheets/FrontendSummary.css'
+
+function FrontendSummary(props) {
+    /*
+    Args:
+        props.json.rotation.success_count (int): Number of successes from rotation averaging.
+        props.json.translation.success_count (int): Number of successes frmo translation averaging.
+        props.json.num_valid_entries (int): Number of valid image pair entries.
+        props.json.num_total_entries (int): Number of total image pair entries.
+        props.json.pose.success_count (int): Number of pose successes.
```

Comment:
I would prefer instead of "Number of pose successes." to say "Number of frames with successfully recovered relative pose"